api4/cloud.go

```diff
@@ -46,11 +46,6 @@ func getSubscription(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if !c.App.SessionHasPermissionTo(*c.App.Session(), model.PERMISSION_SYSCONSOLE_READ_BILLING) {
```

Comment:
Removing this permission you are exposing way too much data to any user. If you want the "remaining users available" I think you should expose only that.

Reply:
Hi Jesùs. Thanks a lot for the review :) This is needed for normal users to access subscription information. From some discussions,  @marianunez said it was alright to remove these permissions.

Reply:
With the your follow up review, I now understand. I could leave these permissions here and shift the part of calculation from the webapp to the new end point.

Reply:
Given that we will now pull the information from the new stats endpoint, we could return the permission back here?